src/base/net/downloadmanager.cpp

```diff
@@ -148,10 +148,20 @@ Net::DownloadManager::DownloadManager(QObject *parent)
         QStringList errorList;
         for (const QSslError &error : errors)
             errorList += error.errorString();
-        LogMsg(tr("Ignoring SSL error, URL: \"%1\", errors: \"%2\"").arg(reply->url().toString(), errorList.join(u". ")), Log::WARNING);
 
-        // Ignore all SSL errors
-        reply->ignoreSslErrors();
+        QString errorMsg;
+        if (Preferences::instance()->isIgnoreSSLErrors())
```

Comment:
It looks like you got the condition mixed up.

Reply:
Good catch. I refactored the function name and the old one meant the reverse. I'll also need to reverse the default value in preference.cpp.